src/main/java/com/powsybl/openloadflow/dc/DcLoadFlowEngine.java

```diff
@@ -102,7 +102,7 @@ public DcLoadFlowResult run(Reporter reporter) {
 
         equationSystem.updateEquations(x);
 
-        this.targetVector = equationSystem.createTargetVector();
+        this.targetVector = TargetVector.createArray(network, equationSystem);
```

Comment:
Shouldn't we do something similar for DC? later maybe. Indeed, I think that the target vector update could be used for sensitivity analyses, where we could avoid to create the vector several times.

Reply:
Maybe it could be interesting but not as much as for AC power flow because for DC power we have just either one or two iteration depending on stack distribution activated or not. For AC power we have often a lot of iterations and target vector number updates, that is why it is in this case very interesting to automate the update.